Build OpenPetra on Appveyor

https://ci.appveyor.com/project/tpokorra/openpetra

currently fails because of missing manifest file or broken path to it:

```
Compiling C:\projects\openpetra\delivery\projects\sharpdevelop4\Ict.Tools.PatchTool.csproj
error CS1926: Error reading Win32 manifest file 'c:\projects\openpetra\csharp\ICT\csharp\ict\buildtools\patchtool\gui\app.manifest' -- 'The system cannot find the path specified. '
```

--- a/inc/nant/OpenPetra.common.xml
+++ b/inc/nant/OpenPetra.common.xml
@@ -246,6 +246,16 @@
 <!-- build tools -->
 <property name="msbuildtask.file" value="${nant::get-base-directory()}/extensions/common/neutral/NAntContrib/NAnt.Contrib.Tasks.dll"
           overwrite="false"/>
+<if test="${not file::exists(msbuildtask.file)}">
+    <!-- for Appveyor -->
+    <property name="msbuildtask.file" value="${path::get-full-path(nant::get-base-directory() + '/../../NAnt.Contrib.Portable.0.92/tools/NAnt.Contrib.Tasks.dll')}"
+          overwrite="true"/>
+    <if test="${not file::exists(msbuildtask.file)}">
+        <property name="msbuildtask.file" value="${path::get-full-path(nant::get-base-directory() + '/../NAnt.Contrib.Portable.0.92/tools/NAnt.Contrib.Tasks.dll')}"
+            overwrite="true"/>
+    </if>
+</if>
+
 <property name="Configuration" value="DEBUG" overwrite="false"/>
 <property name="compileForStandalone" value="false" overwrite="false"/>
           

--- a/inc/nant/OpenPetra.csharp.xml
+++ b/inc/nant/OpenPetra.csharp.xml
@@ -3,6 +3,8 @@
 
 <include buildfile="OpenPetra.default.targets.xml"/>
 
+<loadtasks assembly="${msbuildtask.file}" unless="${task::exists('msbuild')}"/>
+
 <target name="MsBuildTarget">
     <msbuild project="${solution.file}" verbosity="Minimal">
         <property name="Configuration" value="${Configuration}"/>

--- a/inc/nanttasks/CompileProject.cs
+++ b/inc/nanttasks/CompileProject.cs
@@ -4,7 +4,7 @@
 // @Authors:
 //       timop
 //
-// Copyright 2004-2013 by OM International
+// Copyright 2004-2015 by OM International
 //
 // This file is part of OpenPetra.org.
 //
@@ -335,7 +335,8 @@ namespace Ict.Tools.NAntTasks
             if (mainProperties.ContainsKey("ApplicationManifest") && (mainProperties["ApplicationManifest"].Length > 0))
             {
                 parameters.CompilerOptions =
-                    parameters.CompilerOptions.Replace("APPMANIFEST", mainProperties["ApplicationManifest"].Replace("\\", "/"));
+                    parameters.CompilerOptions.Replace("APPMANIFEST",
+                        Path.GetFullPath(Path.GetDirectoryName(FCSProjFile) + "/" + mainProperties["ApplicationManifest"].Replace("\\", "/")));
             }
 
             String FrameworkDLLPath = Path.GetDirectoryName(System.Reflection.Assembly.GetAssembly(typeof(System.Type)).Location);

